sql/src/main/java/io/crate/analyze/AbstractInsertAnalyzer.java

```diff
@@ -120,46 +119,34 @@ private boolean checkReferencesForGeneratedColumn(ColumnIdent columnIdent, Abstr
      *
      * the created column reference is returned
      */
-    protected Reference addColumn(String column, AbstractInsertAnalyzedStatement context, int i) {
-        assert context.tableInfo() != null;
-        return addColumn(new ReferenceIdent(context.tableInfo().ident(), column), context, i);
-    }
-
-    /**
```

Comment:
I removed the whole function-overload. Now there is only one which contains the same comment